modules/openapi-generator/src/main/java/org/openapitools/codegen/utils/ModelUtils.java

```diff
@@ -525,31 +525,6 @@ public static boolean isEmailSchema(Schema schema) {
         return false;
     }
 
-    /**
-     * Check to see if the schema is a model with at least one properties
-     *
-     * @param schema potentially containing a '$ref'
-     * @return true if it's a model with at least one properties
-     */
-    public static boolean isModel(Schema schema) {
```

Comment:
Removing a public helper is a breaking change (these are used by programmatic consumers, and this would therefore need to be deprecated before removal).